tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.settings.app import Settings
+from marionette_driver import By
+
+
+class TestSettingsRTL(GaiaImageCompareTestCase):
+    _page_locator = (By.ID, 'root')
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+    def test_settings_app(self):
+
+        settings = Settings(self.marionette)
+        settings.launch()
+
+        self.take_screenshot('main')
+
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+
+        settings.open_wifi_settings()
```

Comment:
Ah, if we do this way, then it might be difficult to scroll within each of the subpage in the future, because some are scrollable while others are not.  And because of the above issue about being unable to get the total page height, we might have to put if statements within the loop that manually specifies the scroll iterations anyway. So although I much rather prefer to do it your method, I think we have keep it this way for the sake of simplicity.  what do you think?